### PR TITLE
Add multi-file support

### LINKED ISSUE #4: Add multi-file support
This should be as easy as passing an `Object` containing options to `showOpenFilePicker()`. Just creating an issue as a technicality.
https://github.com/SpellCheckJS/html-client/blob/eb48873718454fc6d050732ac437961dd1cc2eb0/browser.js#L10

### PR DESCRIPTION
**Pull request by hello-smile6**
Fixes #4

### CODE REVIEW

**Comment by hello-smile6** [comment]
Seems like I did this right. @webdev03 Could you check?

**Comment by webdev03** [comment]
What is this?

**Comment by hello-smile6** [comment]
> What is this?

An HTML5 client for <a href="https://github.com/SpellCheckJS">SpellCheckJS</a>. I'm trying to add multi-file support.

**Comment by webdev03** [comment]
Ok, I've never heard of SpellCheckJS.

File: browser.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ let main=async function() {
     initButton.innerText="Start";
     initButton.addEventListener("click", function() {
         let script=async function() {
-            let fileRefs=await window.showOpenFilePicker();
+            let fileRefs=await window.showOpenFilePicker({
+                multiple: true
+            });
```
Comment (hello-smile6) [inline browser.js:12]: @webdev03 Here's the section I'm wondering about.

Comment (webdev03) [reply browser.js:12]: Oh Ok

Comment (hello-smile6) [reply browser.js:12]: > Oh Ok

Does that look right?

**Review by imdevarsh — APPROVED** [review]
Probably, although I'm not familiar with file system APIs or opening files.

**Comment by hello-smile6** [comment]
> Probably, although I'm not familiar with file system APIs or opening files.

Okay. I just tested, and it works. I'll merge this. Could you make a NodeJS client?

**Comment by webdev03** [comment]
What do you mean? @hello-smile6

**Comment by hello-smile6** [comment]
> What do you mean? @hello-smile6

Could you make a NodeJS CLI version of this?

**Comment by hello-smile6** [comment]
> Ok, I've never heard of SpellCheckJS.

I'm making it.

**Comment by webdev03** [comment]
CLI version?

**Comment by hello-smile6** [comment]
> CLI version?

Why not?

**Pull request merged**